Neos.Flow/Classes/I18n/EelHelper/TranslationHelper.php

```diff
@@ -109,10 +109,11 @@ public function allowsCallOfMethod($methodName)
     protected function translateByExplicitlyPassedOrderedArguments($id, $originalLabel = null, array $arguments = [], $source = 'Main', $package = null, $quantity = null, $locale = null)
     {
         $translationParameterToken = $this->createTranslationParameterToken($id);
+        $source = isset($source) ? str_replace('.', '/', $source) : 'Main';
```

Comment:
```suggestion
        $source = $source === null ? 'Main' : str_replace('.', '/', $source);
```

Just I tiny nitpick: I'd go four using a null-reference check instead of `isset` here, because imho it's more readable. (`isset` has some tricky semantics. Those are completely irrelevant here, but whenever I see it, I have to think about the implications, while `=== null` tells the story immediatey).